package/CHANGELOG

```diff
@@ -25,10 +25,15 @@ Enhancements
     number density or do not normalize at all. (Issue #3687)
 
 Changes
+  * To add additional optional packages for different file formats when
+    using `pip` use the new "extra_formats" extras_require target as in
+    `pip install ./package[extra_formats]` (Issue #3701)
 
 Deprecations
   * Deprecate `density` parameter in favor of `norm` in InterRDF_s
     (Issue #3687)
+  * The extras_requires target "AMBER" for `pip install ./package[AMBER]`
+    will be removed in 3.0. Use "extra_formats". (Issue #3701, PR #3711)
```

Comment:
```suggestion
  * The extras_requires target "AMBER" for `pip install ./package[AMBER]`
    will be removed in 3.0. Use "extra_formats". (Issue #3701, PR #3711)
  * Deprecate `density` parameter in favor of `norm` in InterRDF_s
    (Issue #3687)
```

Newest entry first :)

Reply:
Also I guess we shouldn't be a pain to the couple of folks that might use this, but does it really need to be a major version breaking change? The number of automated workflow using this should be near 0, and there's no way they'll get a deprecatewarning about this. If we enact the change the biggest issue is that things will just break cleanly right? (or will it just ignore the keyword and go ahead)

Reply:
I don’t think it really needs to be a major revision deprecation. I was just very conservative. I’m not sure how many people used it, tbh.

i haven’t tried proving missing/incorrect target names. I *assume* it will break. 

I’m happy to go with a deprecation for 2.4, too, just let me know or change directly in setup.py and CHANGELOG.

Reply:
I set the deprecation for 2.4.

Reply:
I tried passing the wrong extra (as 'analyses' instead of 'analysis')
```
WARNING: mdanalysis 2.3.0.dev0 does not provide the extra 'analyses'
````

So it won't fail if you give it the wrong thing (which is rather annoying). I'm still ok with setting the deprecation for 2.4 given the fact that we can't suitably warn users anyways, it'll be a problem either way.